cardsort.py

```diff
@@ -135,7 +137,7 @@ def test(self, g):
                 self.game.grid.card_table[self.game.grid.grid.index(r)]\
                     .set_orientation(self.rotation_sets[o][r], False)
             if self.game.test() is True:
-                print((_("You solved the puzzle.")))
+                print(_("You solved the puzzle."))
```

Comment:
@Creatune the external "(" and the "_" is not required in this print statement. Please remove it.

Reply:
I am so sorry. I was confused why the hyphen was there. So I didn't remove it. My humble apologies.

Reply:
Thanks for pointing to it again. Telling you to remove the "_"  and the "()" is a mistake that I made. I didn't paid attention to the fact that the statement should be included in translations.

The _() is used with those string that needs to be translated to other languages by the translation team.

I am sorry for the extra effort, it would be great if you delete your last two commits or revert the changes made in those commits.

Reply:
No problem. At least it was a learning experience for me - which is why I am participating in GCI :) Thanks for mentioning. I'll revert to the previous changes.